integrity-backend/input-proofmode-signal-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,53 @@
+{
+  "contentMetadata":
+  {
+    "name": "Authenticated image",
+    "mime": "image/jpeg",
+    "description": "Image with ProofMode metadata received via Signal",
+    "author":
+    {
+      "type": "Organization",
+      "identifier": "https://starlinglab.org",
+      "name": "Starling Lab"
+    },
+    "dateCreated": "2022-03-29T13:54:20Z",
+    "extras":
+    {},
+    "private":
+    {
+      "proofMode":
```

Comment:
@YurkoWasHere added in your sample data. Note this should be in `private` and has a `proofMode` container. The ZIPs currently have them in `extras`.